Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/FitGaussianPeaksTest.py

```diff
@@ -131,6 +131,24 @@ def test_algorithm_with_negative_max_peak_sigma_throws(self):
                              PeakGuessTable=self.peak_guess_table,
                              MaxPeakSigma=-1.0)
 
+    def test_algorithm_with_even_fit_window_throws(self):
+        with self.assertRaises(Exception):
```

Comment:
Sorry I'm going to be a pain, I've just noticed we're catching Exception instead of the specific Exception we're expecting.

The problem with this is any sort of exception (such as IndentationError/OutOfMemoryError ..etc.) would pass this test, rather than the exception we're expecting (e.g. RuntimeError)